File: app/src/main/java/kandroid/data/UseItem.java
package kandroid.data;

import kandroid.observer.POJO;
import kandroid.utils.Identifiable;

public class UseItem extends Data<UseItem.ApiUseitem> implements Identifiable{

    public int getItemID() {
        return data.api_id;
    }

    public int getCount() {
        return data.api_count;
    }

    @Override
    public int getID() {
        return data.api_id;
    }

    public static class ApiUseitem extends POJO {
        public int api_id;
        public int api_count;
    }
}

File: app/src/main/java/kandroid/data/UseItem.kt
package kandroid.data

import kandroid.data.JsonWrapper
import kandroid.utils.Identifiable
import kandroid.utils.json.get
import kandroid.utils.json.int

class UseItem : JsonWrapper(), Identifiable {

    val itemId: Int get() = data["api_id"].int()
    val count: Int get() = data["api_count"].int()

    override val id: Int get() = itemId
}